core/app/models/concerns/spree/user_reporting.rb

```diff
@@ -1,22 +1,24 @@
 module Spree
   module UserReporting
-    extend DisplayMoney
-    money_methods :lifetime_value, :average_order_value
 
-    def lifetime_value
-      orders.complete.sum(:total)
+    def display_lifetime_value(store)
+      Spree::Money.new(lifetime_value(store), currency: store.default_currency)
     end
 
-    def order_count
-      orders.complete.size
+    def lifetime_value(store)
```

Comment:
```suggestion
    def lifetime_value(store = nil)
      store ||= Store.default
```

so this won't break any previous spree versions

Reply:
done